### ISSUE TITLE
Mixed delimiters in header of baselineLD_v1.1 annotation

### ISSUE THREAD

**Issue by b-dawes** [bulik/ldsc#105]
I was using the baseline model from the file:
https://data.broadinstitute.org/alkesgroup/LDSCORE/1000G_Phase3_baselineLD_v1.1_ldscores.tgz

I ran into an issue when parsing the annot.gz files and found it was because the header lines mix tabs and whitespaces. Specifically, everything is tab delimited except for the MAFbin annotations which are delimited with single spaces.

This is only present in the header, all the other lines are entirely tab delimited.

**Comment by rkwalters** [comment]
Hi,
Confirmed I see this as well. It’s not an issue for ldsc since it only assumes those files are whitespace delimited (i.e. doesn’t differentiate tabs vs. spaces) but I can see how that could be an annoyance for other applications.

Hilary, would it be possible to update these files?

Cheers,
Raymond

**Comment by b-dawes** [comment]
Yes, I was able to run LDSC on these files with no issues. I only ran into this because I was trying to load the files into R to check the intersection of our annotation files with the baseline annotations.

It's not a big issue, but I figured I would post and let you know about it.

**Comment by rkwalters** [comment]
Hi,
After some internal discussion, it appears there consensus to leave the existing `baselineLD_v1.1` file as is for the sake of documenting the exact reference file used in the corresponding papers. We will however try to make sure this is fixed in future updates to avoid this annoyance when working with precomputed LD scores for future models.
Cheers,
Raymond
